src/app/core/components/item-nav/item-nav.component.ts

```diff
@@ -126,11 +126,13 @@ export class ItemNavComponent implements OnInit, OnDestroy {
   ngOnInit() {
 
     this.subscriptions.push(
+
+      // This first subscription only follow change in the current item id and use switch map to cancel previous requests
       this.currentContent.currentContent$.pipe(
 
         // we are only interested in items
         map(content => (content !== null && isItemInfo(content) ? content.data : null)),
-        distinctUntilChanged((v1, v2) => v1 === null && v2 === null), // only to avoid sending multiple null
+        distinctUntilChanged((v1, v2) => v1 === null && v2 === null || ( v1 !== null && v2 !== null && v1.nav.itemId === v2.nav.itemId)),
```

Comment:
The comment on this line was deleted, considering the argument of the `distinctUntilChanged` function became more complicated, it may be usefull to bring it back.

Reply:
but it does not do the same. 
Ok I'll re-add another comment